x/ccv/provider/keeper/relay.go

```diff
@@ -217,17 +219,18 @@ func (k Keeper) SendVSCPacketsToChain(ctx sdk.Context, chainID, channelID string
 func (k Keeper) QueueVSCPackets(ctx sdk.Context) {
 	valUpdateID := k.GetValidatorSetUpdateId(ctx) // current valset update ID
 
-	// get the bonded validators from the staking module
```

Comment:
This docstring is still relevant. Explain better why `bondedValidators` is needed and why `consensusValidators` is needed (i.e., for TopN chains).